README.md

```diff
@@ -276,81 +278,383 @@ schedule is listed as N/A.
+be expected once every 46 lookup operations.
+
+##### Fine-tuning: Fence-Pointer Index Type <span id="fine_tuning_fence_pointer_index_type" class="anchor"></span>
+
+The configuration parameter `confFencePointerIndex` affects the type and
+size of the fence-pointer indexes. Tables maintain a fence-pointer index
+in memory for each run on disk. These fence-pointer indexes store the
+keys at the boundaries of each page of memory to ensure that each lookup
+has to read at most one page of memory from each run. Tables support two
+types of fence-pointer indexes:
+
+`OrdinaryIndex`  
+Ordinary indexes are designed for any use case.
+
+Ordinary indexes store one serialised key per page of memory. The total
+in-memory size of all indexes is $`K \cdot \frac{n}{P}`$ bits, where
+$`K`$ refers to the average size of a serialised key in bits.
+
+`CompactIndex`  
+Compact indexes are designed for the use case where the keys in the
+table are uniformly distributed, such as when using hashes.
+
+Compact indexes store the 64 most significant bits of the minimum
+serialised key of each page of memory. This requires that serialised
+keys are *at least* 64 bits in size. Compact indexes store 1 additional
+bit per page of memory to resolve collisions, 1 additional bit per page
+of memory to mark entries that are larger than one page, and a
+negligible amount of memory for tie breakers. The total in-memory size
+of all indexes is $`66 \cdot \frac{n}{P}`$ bits.
+
+##### Fine-tuning: Disk Cache Policy <span id="fine_tuning_disk_cache_policy" class="anchor"></span>
+
+The configuration parameter `confDiskCachePolicy` determines how the
+database uses the OS page cache. This may improve performance if the
+database's *access pattern* has good *temporal locality* or good
+*spatial locality*. The database's access pattern refers to the pattern
+by which entries are accessed by lookup operations. An access pattern
+has good temporal locality if it is likely to access entries that were
+recently inserted. An access pattern has good spatial locality if it is
+likely to access entries that have nearby keys.
+
+- Use the `DiskCacheAll` policy if the database's access pattern has
+  either good spatial locality or both good spatial and temporal
+  locality.
+
+- Use the `DiskCacheLevelOneTo l` policy if the database's access
+  pattern only has good temporal locality. The variable `l` determines
+  the number of levels that are cached. For a description of levels, see
+  [Merge Policy, Size Ratio, and Write Buffer
+  Size](#fine_tuning_data_layout "#fine_tuning_data_layout"). With this
+  setting, the database can be expected to cache up to $`\frac{k}{P}`$
+  pages of memory, where $`k`$ refers to the number of entries that fit
+  in levels $`[1,l]`$ and is defined as $`\sum_{i=1}^{l}BT^{i}`$.
+
+- Use the `DiskCacheNone` policy if the database's access pattern has
+  does not have good spatial or temporal locality. For instance, if the
+  access pattern is uniformly random.
```

Comment:
This is good, but on reflection I think I didn't describe it to you quite perfectly.

I think common meaning would say that good temporal locality covers both entries that were recently inserted, or entries that were recently looked up.

`DiskCacheAll` helps with temporal locality for inserts and lookups, however `DiskCacheLevelOneTo` only helps with temporal locality for inserts. `DiskCacheLevelOneTo` does not help with temporal locality for lookups.

So unfortunately, this complicates your presentation somewhat.